neard/src/cli.rs

```diff
@@ -263,8 +264,20 @@ impl RunCmd {
         let sys = actix::System::new();
         sys.block_on(async move {
             nearcore::start_with_config(home_dir, near_config);
-            tokio::signal::ctrl_c().await.unwrap();
-            info!("Got Ctrl+C, stopping");
+
+            let sig = if cfg!(unix) {
+                use tokio::signal::unix::{signal, SignalKind};
+                let mut sigint = signal(SignalKind::interrupt()).unwrap();
+                let mut sigterm = signal(SignalKind::terminate()).unwrap();
+                futures::select! {
+                    _ = sigint .recv().fuse() => "SIGINT",
+                    _ = sigterm.recv().fuse() => "SIGTERM"
+                }
+            } else {
+                tokio::signal::ctrl_c().await.unwrap();
+                "Ctrl+C"
+            };
+            info!("Got {}, stopping", sig);
```

Comment:
```suggestion
            info!(target: "neard", "Got {}, stopping", sig);
```

we always specify the target everywhere else: https://github.com/near/nearcore/blob/master/docs/architecture.md#logging--observability